Add definition and perspective for 'KPI' (#212)
cherryontech/website: src/assets/content/dictionary/kpi.md

---
title: KPI
definition: Key Performance Indicators (KPIs) is a critical (key) indicator of progress toward an intended result. A KPI provides a focus for strategic and operational improvement, create an analytical basis for decision making and help focus attention on what matters most.
sources:
- sourceurl: https://kpi.org/KPI-Basics
perspectives:

> - meaning: a metric or indicator to measure the work done by software teams in order to remain accountable to set objectives. Eg lines of code, bugs, task completion dealines

Reviewer: ```suggestion
- meaning: a way to measure the work I have done in order to remain accountable to set objectives
```
